server/routes/journeys/record-review/e2e.cy.ts

```diff
@@ -0,0 +1,132 @@
+
+    cy.url().should('to.match', /\/record-review$/)
+
+    cy.findByRole('link', { name: 'Check and save report' }).should('not.exist')
+    cy.findAllByText('Incomplete').should('have.length', 3)
+    cy.findByText('Cannot save yet').should('be.visible')
+
+    cy.findByRole('link', { name: /Details of the review/ }).click()
+    cy.findByRole('textbox', { name: 'Enter details of the review' }).clear().type('review details', { delay: 0 })
+    getContinueButton().click()
+    cy.url().should('to.match', /record-review$/)
+
+    cy.findByRole('link', { name: /Participants and contributions/ }).click()
+    cy.findByRole('button', { name: 'Add participant' }).click()
+    cy.url().should('to.match', /\/participant-contribution-details\/1$/)
+    cy.findByRole('radio', { name: /yes/i }).click()
+    cy.findByRole('textbox', { name: /What’s the participant’s name\?/ })
+      .clear()
+      .type('John Smith', { delay: 0 })
+    cy.findByRole('textbox', { name: /What’s their role\?/ })
+      .clear()
+      .type('a role', { delay: 0 })
+    cy.findByRole('textbox', { name: /What did they contribute to the review\? \(optional\)/ })
+      .clear()
+      .type('a contrib', { delay: 0 })
+    getContinueButton().click()
+    getContinueButton().click()
+    cy.url().should('to.match', /record-review$/)
+
+    cy.findByRole('link', { name: /Outcome/ }).click()
+    cy.findByRole('radio', { name: 'What’s the outcome of this review? Keep the prisoner on the plan' }).click()
+    getContinueButton().click()
+    cy.url().should('to.match', /\/next-review-date$/)
+    cy.findByRole('textbox', { name: "When will you next review the plan with Tes'name User?" }).type(
+      formatDate(new Date(), 'dd/MM/yyyy'),
+      {
+        delay: 0,
+      },
+    )
+    getContinueButton().click()
+    cy.url().should('to.match', /\/record-review$/)
+
+    cy.findByRole('link', { name: /Check and save report/ }).click()
+    cy.url().should('to.match', /\/check-answers$/)
+
+    verifyCya()
+  })
+
+  const verifyCya = () => {
+    cy.contains('dt', 'Review details').next().should('include.text', 'review details')
+    cy.contains('dt', 'Review outcome').next().should('include.text', 'Keep the prisoner on the plan')
+    cy.contains('dt', 'Next review date').next().should('include.text', formatDate(new Date(), 'd MMMM yyyy'))
+    cy.contains('dt', 'Name').next().should('include.text', 'John Smith')
+    cy.contains('dt', 'Role').next().should('include.text', 'a role')
+    cy.contains('dt', 'Attended in person').next().should('include.text', 'Yes')
+    cy.contains('dt', 'Contribution').next().should('include.text', 'a contrib')
+
+    verifyCyaChangeLinks()
+  }
+
+  const verifyCyaChangeLinks = () => {
+    cy.findByRole('link', { name: /Change the details of the review/ }).click()
+    cy.url().should('to.match', /details#summary$/)
+    cy.get('textarea').clear().type('changed summary', { delay: 0 })
```

Comment:
Is it worth using `cy.focused()` for these just as a sanity check that the input element is being focused?

Reply:
@JackReeveMoJ Why do we care about the text area being focused when we type? Im confused 😅

Reply:
cya page a/c says something about focusing on the specific html input element.
in the unit-esque cypress test for just the check-answers page, we didn't test its interactions with the other pages. 
so i guess that would belong to the full e2e test here if we're concerned about that.

Reply:
Oh you mean when we click the change link - that sort of detail really should belong in the check-answers tests, not this one. Added there :)